### PR TITLE
Fix issue #954, add missing ReportOptionsDialog PageLayout call

### PR DESCRIPTION
**Pull request by arnost00**

### CODE REVIEW

File: quickevent/app/quickevent/plugins/Runs/src/runswidget.cpp [hunk 1]
```diff
@@ -220,8 +220,21 @@ void RunsWidget::settleDownInPartWidget(quickevent::gui::PartWidget *part_widget
 		auto *a = new qfw::Action(tr("&Competitors with rented cards"));
 		connect(a, &qfw::Action::triggered, [this]() {
 			qff::MainWindow *fwk = qff::MainWindow::frameWork();
+			quickevent::gui::ReportOptionsDialog dlg(fwk);
```
Comment (fvacek) [inline quickevent/app/quickevent/plugins/Runs/src/runswidget.cpp:223]: please, bump QE version

**Review by fvacek — CHANGES_REQUESTED** [review]

File: quickevent/app/quickevent/plugins/Runs/src/runswidget.cpp (hunk 1 above)
Comment (arnost00) [reply quickevent/app/quickevent/plugins/Runs/src/runswidget.cpp:223]: No problem.
For this PR and PR #956 together or for both independently?

Comment (fvacek) [reply quickevent/app/quickevent/plugins/Runs/src/runswidget.cpp:223]: for both pls

Comment (arnost00) [reply quickevent/app/quickevent/plugins/Runs/src/runswidget.cpp:223]: Done

**Pull request merged**